crates/core/src/rpc/surfnet_cheatcodes.rs

```diff
@@ -452,4 +465,103 @@ impl SvmTricksRpc for SurfnetCheatcodesRpc {
             })
         });
     }
+
+    /// Clones a program account from one program ID to another.
+    /// A program account contains a pointer to a program data account, which is a PDA derived from the program ID.
+    /// So, when cloning a program account, we need to clone the program data account as well.
+    ///
+    /// This method will:
+    ///  1. Get the program account for the source program ID.
+    ///  2. Get the program data account for the source program ID.
+    ///  3. Calculate the program data address for the destination program ID.
+    ///  4. Set the destination program account's data to point to the calculated destination program address.
+    ///  5. Copy the source program data account to the destination program data account.
+    fn clone_program_account(
```

Comment:
Could we conform to the doc convention we have in the other RPC methods? 🙏

Reply:
This is a doc comment on the implementation, not on the trait, so these are our internal comments rather than the public RPC docs.

We don't have docs on the cheatcode RPCs yet, you cool if we follow on in another PR?